packages/xstate-test/test/index.test.ts

```diff
@@ -1,4 +1,4 @@
-import { assign, createMachine } from 'xstate';
+import { EventFrom, assign, createMachine } from 'xstate';
```

Comment:
With [the proposed change](https://github.com/statelyai/xstate/pull/3864/files#r1123172455) this import becomes obsolete
```suggestion
import { assign, createMachine } from 'xstate';
```